core/src/main/java/com/datastax/dse/driver/api/core/auth/DseGssApiAuthProviderBase.java

```diff
@@ -182,6 +183,17 @@ public Builder withLoginConfiguration(@Nullable Configuration loginConfiguration
         this.loginConfiguration = loginConfiguration;
         return this;
       }
+      /**
+       * Sets a login configuration that will be used to create a {@link LoginContext}.
+       *
+       * <p>You MUST call either this method or {@link #withSubject(Subject)}; if both are called,
+       * the subject takes precedence, and the login configuration will be ignored.
```

Comment:
Nit: there are now 3 different methods that can be called. But I think this will be clear enough, trying to reformulate it with 3 methods makes it more confusing.